### PR TITLE
upgrade omero-install

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This PR upgrades omero-install

#62 merged to speed up testing

tasks:
- tag https://github.com/ome/omero-install/pull/129
- propagate tag to https://github.com/openmicroscopy/devslave-c7-docker/pull/9.
  - Tested container via https://github.com/aleksandra-tarkowska/devslave-c7-docker/commit/4f95a4fd603dc84ae31c1908a4a5c3ecd786476a, temp release on https://hub.docker.com/r/olat/devslave-c7-docker/builds/
- replace https://github.com/openmicroscopy/devspace/pull/63/commits/f166872736273fce3c8a4acf90f4646eb6857573 in this PR with the official image

### CODE REVIEW

**Comment by aleksandra-tarkowska** [comment]
👍 https://10.0.51.143:8443 is blue

**Comment by aleksandra-tarkowska** [comment]
using https://github.com/openmicroscopy/infrastructure/pull/240 with vars:

```
devspace_omero_branch: develop
snoopy_dir_path: "/path/to/snoopy"
devspace_git_repo: "https://github.com/aleksandra-tarkowska/devspace.git"
devspace_git_repo_version: "omero_install_525"
```

successfully played devspace:

```
ola$ ansible-playbook -l devspace-test-030 -u centos devspace.yml -vvv 

PLAYBOOK: devspace.yml *********************************************************
...
RUNNING HANDLER [devspace : assert] ********************************************
ok: [4f931990-f15a-4fe3-8c86-8a350590ba90] => {"changed": false, "invocation": {"module_args": {"that": ["jenkins.devspace_jenkins_1.state.running", "pg.devspace_pg_1.state.running", "testintegration.devspace_testintegration_1.state.running", "omero.devspace_omero_1.state.running", "web.devspace_web_1.state.running", "nginx.devspace_nginx_1.state.running", "nginxjenkins.devspace_nginxjenkins_1.state.running", "redis.devspace_redis_1.state.running", "seleniumhub.devspace_seleniumhub_1.state.running", "seleniumfirefox.devspace_seleniumfirefox_1.state.running", "seleniumchrome.devspace_seleniumchrome_1.state.running"]}, "module_name": "assert"}, "msg": "all assertions passed"}

PLAY RECAP *********************************************************************
4f931990-f15a-4fe3-8c86-8a350590ba90 : ok=37   changed=28   unreachable=0    failed=0   
```

File: home/jobs/BIOFORMATS-ant/config.xml [hunk 1]
```diff
@@ -22,12 +22,12 @@
...
           <name>testng.runWriterSaveBytesTests</name>
           <description></description>
-          <defaultValue>100</defaultValue>
+          <defaultValue>50</defaultValue>
         </hudson.model.StringParameterDefinition>
         <hudson.model.StringParameterDefinition>
           <name>testng.runWriterTilingTests</name>
           <description></description>
-          <defaultValue>100</defaultValue>
+          <defaultValue>2</defaultValue>
```
Comment (aleksandra-tarkowska) [inline home/jobs/BIOFORMATS-ant/config.xml]: reduced as discussed

**Comment by joshmoore** [comment]
@sbesson proposes a breaking bump. `0.4.0`? (for comparison, `0.3.0` was the move to ansible)

**Comment by joshmoore** [comment]
Merging & tagging (on zoom with @sbesson). One general thought is that some of the scripts could eventually be migrated out to another repository (though not management_tools) for re-use in ci-master. ("ci-scripts"?). I'm also unsure what the `idr-ci` strategy will be sense we now have an OMERO 5.3 dependency. When/if the time comes, we'll handle that though.

Thanks.

**Pull request merged**